app.py

```
@@ -171,11 +173,271 @@ def render_settings(action):
+        return jsonify({'success': False, 'error': str(e)})
+
+
 @app.errorhandler(Exception)
 def handle_exception(e):
     print(e)
     return render_template("error.html", error=e), 500
 
 
 if __name__ == '__main__':
-    app.run(host='0.0.0.0', use_debugger=False)
+    app.run(host='0.0.0.0', use_debugger=True)
```

Ruff (0.15.4)

[error] Possible binding to all interfaces

(S104)